manager.php

```diff
@@ -27,6 +27,11 @@
         	<input type="submit" value="Find Employee" name="findEmployee"></p>		
 	</div>
 	
+
+       	<input type="submit" value="Find Great Customers" name="findCust"></p>	
```

Comment:
Find Great Customers returns no results with a refreshed db, you should add customers that are 'great' (which i'm not sure what that means just reading at the button description but hey) to the sql file